api/validators/GenericRequests.ts

```diff
@@ -8,3 +8,18 @@ export class Pagination implements IPagination {
   @Min(0)
   limit?: number;
 }
+
+export class ValidEmail {
+  @IsEmail()
+  email:string;
+}
+export class ValidUuid {
+  @IsUUID()
+  uuid:Uuid;
+}
+
+export class ValidAccessCode {
+  @IsHexadecimal()
+  @Length(32, 32)
+  accessCode:string;
+}
```

Comment:
nit: with our validation checks now I think it's implied that all uuids/emails/accessCodes that are passed in are "valid", so we don't need to mention that within the class name. Something like `EmailParam` or `UuidParam` might make it more clear the intention of the classes